Add configuration instructions for router wire logs in choreo connect (#5943)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/configure-logs-router.md

'x-envoy-decorator-operation', 'localhost:^/v2/pet/findByStatus(\?([^/]+))?$'
```

Follow [command line options]({{envoy_path}}/operations/cli) for more information.

## Router wire logs

> By enabling wire logs in the router component, you can inspect headers, body, and trailers of the request/response. This is helpful to see the changes applied to the request or response after request path and/or response path interceptors.

Reviewer: ```suggestion
By enabling wire logs in the Router component, you can inspect headers, body, and trailers of the request/response. This is helpful to see the changes applied to the request or response after request path and/or response path interceptors.
```

> To enable the wire logs add following configuration to `log_config.toml` file in [these directories]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/configure-logs-overview/#log_config_toml) depending on your Choreo Connect deployment setup.

Reviewer: ```suggestion
To enable the wire logs, add the following configuration to the `log_config.toml` file in [these directories]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/configure-logs-overview/#log_config_toml) depending on your Choreo Connect deployment setup.
```

```yaml
[wireLogs]
enable = true
include = ["Headers", "Body", "Trailers"]
```

You need to set `enable` property as `true` as well as keep at least one from the set {"Headers", "Body", "Trailers"} depending on which parts you want to inspect in the `include` array.

> An API request invoked at Choreo Connect with the above wire log configuration will generate logs like below.

Reviewer: ```suggestion
An API request invoked at Choreo Connect with the above wire log configuration will generate logs similar to the following.
```
